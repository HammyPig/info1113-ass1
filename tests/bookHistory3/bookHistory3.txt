deals with repeated renting and relinquishing