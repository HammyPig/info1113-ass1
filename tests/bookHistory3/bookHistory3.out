user: Success.

user: Success.

user: 17 books successfully added.

user: No such book in system.

user: No rental history.

user: Success.

user: No rental history.

user: Success.

user: 100000

user: Success.

user: Success.

user: 100000
100001

user: Success.

user: 100000
100001

user: Success.

user: 100000
100001
100001

user: Success.

user: Success.

user: 100000
100001
100001
100000

user: Ending Library process.
